insights/parsers/df.py

```diff
@@ -160,6 +156,23 @@ def get_mount(self, name):
         """Record: Returns Record obj for mount point ``name``."""
         return self.mounts.get(name)
 
+    def get_dir(self, path):
```

Comment:
It's a useful method, after it's merged, I can replace the `get_mount_fs` in sat_diskfree_check with this one.